scripts/wheel-rpc-server.py

```diff
@@ -98,25 +98,37 @@ def run(self):
         pack("src/deltachat_rpc_server/__init__.py", "")
 
 
-def build_wheel(version, binary, tag):
+def build_wheel(version, binary, tag, windows=False):
     filename = f"dist/deltachat_rpc_server-{version}-{tag}.whl"
 
     with WheelFile(filename, "w") as wheel:
         wheel.write("LICENSE", "deltachat_rpc_server/LICENSE")
         wheel.write("deltachat-rpc-server/README.md", "deltachat_rpc_server/README.md")
-        wheel.writestr(
-            "deltachat_rpc_server/__init__.py",
-            """import os, sys
+        if windows:
+            wheel.writestr(
+                "deltachat_rpc_server/__init__.py",
+                """import os, sys, subprocess
+def main():
+    argv = [os.path.join(os.path.dirname(__file__), "deltachat-rpc-server.exe"), *sys.argv[1:]]
```

Comment:
I think it is better just to check if os.name == "nt" add the .exe prefix, so it is the same script for all platforms without needing extra window arg